### PR TITLE
Add moonbeam deployment

### PR DESCRIPTION
**Pull request by rmeissner**
RPC endpoints for verification:

- 1285 (Moonriver): `https://rpc.moonriver.moonbeam.network`
- 1287 (Moonbase Alpha): `https://rpc.testnet.moonbeam.network`

### CODE REVIEW

File: src/assets/v1.3.0/sign_message_lib.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "100": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
     "137": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
     "246": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "1287": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```
Comment (rmeissner) [inline src/assets/v1.3.0/sign_message_lib.json:15]: This is the latest version, the others should be updated in a separate PR

**Review by jpalvarezl — APPROVED** [review]

File: src/assets/v1.3.0/sign_message_lib.json (hunk 1 above)
Comment (rmeissner) [reply src/assets/v1.3.0/sign_message_lib.json:16]: The moonriver deployments are part of this PR too. The PR has been merged and you can see the latest changes on the main branch. We will also release this soon to npm.

**Comment by datradito** [comment]
Hi @rmeissner , do we have the contracts verified on etherscan? to be sure if the scripts:
```
yarn hardhat --network <network> etherscan-verify yarn hardhat --network <network> local-verify
```
 were executed. Thanks.
